### PR TITLE
Added missing "!" in function canUseAbility and changed clickRate to 20

### PR DESCRIPTION
**Pull request by K4r0qtuYNE5G4qgZ**

### CODE REVIEW

File: autoPlay.user.js [hunk 1]
```diff
@@ -16,7 +16,7 @@
 	"use strict";
 
 // OPTIONS
-var clickRate = 10;
+var clickRate = 20;
```
Comment (xPaw) [inline autoPlay.user.js:19]: This is an unrelated change to your pull request.

Comment (K4r0qtuYNE5G4qgZ) [inline autoPlay.user.js:19]: Simply forgot to mention it in File comment but added it to Title, also i not really wanted to fork it, happened by accident, but then decided to fix that Bug and to try that pull Thing. ;)

Also i changed the cickRate to default 20 because i'm tired of changed it everytime, 20 works well...
